testgrid/config/config.yaml

```diff
@@ -1685,80 +1673,89 @@ test_groups:
 #
 
 dashboards:
-- name: canonical
+- name: canonical-ubuntudev-k8sdev
   dashboard_tab:
-  - name: gce-ubuntu-node
-    test_group_name: ci-kubernetes-e2e-gce-ubuntu-node
-  - name: gce-ubuntu-node-serial
-    test_group_name: ci-kubernetes-e2e-gce-ubuntu-node-serial
-  - name: gce-ubuntu-1.7-node
-    test_group_name: ci-kubernetes-e2e-gce-ubuntu-1-7-node
-  - name: gce-ubuntu-1.7-node-serial
-    test_group_name: ci-kubernetes-e2e-gce-ubuntu-1-7-node-serial
-  - name: gce-ubuntu-1.6-node
-    test_group_name: ci-kubernetes-e2e-gce-ubuntu-1-6-node
-  - name: gce-ubuntu-1.6-node-serial
-    test_group_name: ci-kubernetes-e2e-gce-ubuntu-1-6-node-serial
   - name: e2enode-ubuntudev-k8sdev-default(gkespec)
     test_group_name: ci-kubernetes-e2enode-ubuntudev-k8sdev-gkespec
   - name: e2enode-ubuntudev-k8sdev-serial
     test_group_name: ci-kubernetes-e2enode-ubuntudev-k8sdev-serial
-  - name: e2enode-ubuntudev-k8sstable1-default(gkespec)
-    test_group_name: ci-kubernetes-e2enode-ubuntudev-k8sstable1-gkespec
-  - name: e2enode-ubuntudev-k8sstable1-serial
-    test_group_name: ci-kubernetes-e2enode-ubuntudev-k8sstable1-serial
-  - name: e2enode-ubuntudev-k8sstable2-default(gkespec)
-    test_group_name: ci-kubernetes-e2enode-ubuntudev-k8sstable2-gkespec
-  - name: e2enode-ubuntudev-k8sstable2-serial
-    test_group_name: ci-kubernetes-e2enode-ubuntudev-k8sstable2-serial
-  - name: e2enode-ubuntustable1-k8sdev-default(gkespec)
-    test_group_name: ci-kubernetes-e2enode-ubuntustable1-k8sdev-gkespec
-  - name: e2enode-ubuntustable1-k8sdev-serial
-    test_group_name: ci-kubernetes-e2enode-ubuntustable1-k8sdev-serial
-  - name: e2enode-ubuntustable1-k8sstable1-default(gkespec)
-    test_group_name: ci-kubernetes-e2enode-ubuntustable1-k8sstable1-gkespec
-  - name: e2enode-ubuntustable1-k8sstable1-serial
-    test_group_name: ci-kubernetes-e2enode-ubuntustable1-k8sstable1-serial
-  - name: e2enode-ubuntustable1-k8sstable2-default(gkespec)
-    test_group_name: ci-kubernetes-e2enode-ubuntustable1-k8sstable2-gkespec
-  - name: e2enode-ubuntustable1-k8sstable2-serial
-    test_group_name: ci-kubernetes-e2enode-ubuntustable1-k8sstable2-serial
   - name: gce-ubuntudev-k8sdev-default
     test_group_name: ci-kubernetes-e2e-gce-ubuntudev-k8sdev-default
   - name: gce-ubuntudev-k8sdev-serial
     test_group_name: ci-kubernetes-e2e-gce-ubuntudev-k8sdev-serial
   - name: gce-ubuntudev-k8sdev-slow
     test_group_name: ci-kubernetes-e2e-gce-ubuntudev-k8sdev-slow
+
+- name: canonical-ubuntudev-k8sbeta
```

Comment:
bikeshed: I feel like the display name is better with explicit version, like ubuntu-vfoo-k8s-v1.8? WDYT?

Reply:
I feel like the versions should be displayed somewhere else other than in the name because the same test job may run different image/k8s versions at different times, and we also don't want to change the name in testgrid every time we update the version mappings.